pkg/archive/archive_test.go

```diff
@@ -724,6 +725,48 @@ func TestTarUntar(t *testing.T) {
 	}
 }
 
+func TestTarAppenderChownOption(t *testing.T) {
+	origin, err := ioutil.TempDir("", "docker-test-tar-chown-opt")
+	if err != nil {
+		t.Fatal(err)
```

Comment:
If you are already using `assert/require` in this test then use it everywhere. This is equal to `require.NoError(t, err)`